sql/core/src/test/scala/org/apache/spark/sql/util/DataFrameCallbackSuite.scala

```diff
@@ -227,9 +227,7 @@ class DataFrameCallbackSuite extends QueryTest
         spark.range(10).select($"id", $"id").write.insertInto("tab")
```

Comment:
can we change this test to trigger a runtime exception? e.g. the column is int type and the input data is string type.

Reply:
OK